src/mad_spark_alt/evolution/cost_estimator.py

```
@@ -0,0 +1,658 @@
+            self._model_costs[model] = ModelCosts(
+                input_cost_per_1k_tokens=costs["input"],
+                output_cost_per_1k_tokens=costs["output"],
+            )
+
+    def estimate_evolution_cost(
+        self,
+        config: EvolutionConfig,
+        model: str = "gpt-4",
+        avg_tokens_per_evaluation: int = 1000,
+        cache_hit_rate: float = DEFAULT_CACHE_HIT_RATE,
+        enable_llm_operators: bool = False,
```

Ruff (0.12.2)

Boolean-typed positional argument in function definition

(FBT001)